ui/admin/app/components/form/host-catalog/index.hbs

```diff
@@ -45,12 +45,12 @@
       </field.errors>
     {{/if}}
   </form.textarea>
-
-  <form.actions
-    @disabled={{if @model.cannotSave @model.cannotSave}}
-    @enableEditText={{t 'actions.edit-form'}}
-    @submitText={{t 'actions.save'}}
-    @cancelText={{t 'actions.cancel'}}
-  />
-
+  {{#if (can 'save model' @model)}}
```

Comment:
I am missing an else case and wondering if the user has no save capabilities, it just sees the form with no actions? Perhaps the else is not need here?

Reply:
oops, seems like my changes didn't get pushed. It needs an else case.

Reply:
yes, we don't need the else case here. The user can't edit the form if they don't have `update` capability.  `save model` is a composite that includes `update`.  see discussion [here](https://github.com/hashicorp/boundary-ui/pull/689)

Reply:
Nice! I miss that discussion